test/sav/sav_compare.c

```diff
@@ -356,8 +356,12 @@ static int is_exception_image_grid(int global_offset)
     if ((v1 == 734 && is_between(v2, 743, 770)) || (v2 == 734 && is_between(v1, 743, 770))) {
         return 1;
     }
-    // road to granary: in julius the roads lead into the granary
-    if (both_between(v1, v2, 591, 657)) {
+    // road to granary: in julius the dirt roads lead into the granary
```

Comment:
The other way around 😁  640-657 are dirt roads

Reply:
How do you know that??? I'll try to implement this exception only for adjacent granaries

Reply:
Use SGReader. If you hover over an image in the left tree, the tooltip will give you "ID 123" and some other info. If you open up C3.sg2, then expand "Land2a.bmp", scroll down to image 44, you'll see that that's the first road tile. It's a paved road and the tooltip says "ID: 591". Scroll down to the first dirt road at image 93, it's ID 640.

Reply:
Never thought about using sgreader... Thanks!